pom.xml

```diff
@@ -366,17 +366,30 @@
     </resources>
     <plugins>
       <plugin>
-        <groupId>org.apache.tomcat.maven</groupId>
-        <artifactId>tomcat7-maven-plugin</artifactId>
-        <version>2.2</version>
+        <groupId>org.codehaus.cargo</groupId>
+        <artifactId>cargo-maven2-plugin</artifactId>
+        <version>1.7.1</version>
         <configuration>
-          <path>/</path>
-          <systemProperties>
-            <java.util.logging.manager>org.apache.juli.ClassLoaderLogManager</java.util.logging.manager>
-            <org.apache.tomcat.util.buf.UDecoder.ALLOW_ENCODED_SLASH>true
-            </org.apache.tomcat.util.buf.UDecoder.ALLOW_ENCODED_SLASH>
-            <java.net.preferIPv4Stack>true</java.net.preferIPv4Stack>
```

Comment:
These properties are safe to remove?

Reply:
They are only used for the `tomcat7-maven-plugin` in local dev environments. Sadly there is no `tomcat8-maven-plugin` so I am switching to cargo. However, they should probably be set in cargo.